src/app/modules/sgid/sgid.controller.ts

```diff
@@ -46,6 +46,32 @@ export const handleLogin: ControllerHandler<
   }
 
   const form = formResult.value
+  // @TODO move this to separate route.
```

Comment:
If this is TODO for subsequent PR, can open an issue and then label here, e.g. // TODO(#XXXX): Move this to separate route

Reply:
I think I'll remove this TODO. I don't think it should be in a separate route. They are logically a little different but most of the code is shared.